modules/react/modal/stories/examples/FullOverflow.tsx

```diff
@@ -1,52 +1,91 @@
+              The above copyright notice and this permission notice shall be included in all copies
+              or substantial portions of the Software.
+            </p>
+            <p>
+              THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+              INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+              PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+              HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+              CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
+              OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua. Amet massa vitae tortor condimentum
+              lacinia quis. Fermentum odio eu feugiat pretium nibh ipsum consequat nisl. Sed lectus
+              vestibulum mattis ullamcorper velit sed. Rutrum tellus pellentesque eu tincidunt
+              tortor aliquam nulla. Vitae turpis massa sed elementum tempus egestas sed sed risus.
+              Cursus vitae congue mauris rhoncus aenean vel elit scelerisque mauris. Id neque
+              aliquam vestibulum morbi blandit cursus risus at. Vel eros donec ac odio tempor orci.
+              Ac felis donec et odio pellentesque diam volutpat. Laoreet non curabitur gravida arcu
+              ac tortor dignissim. Rhoncus urna neque viverra justo nec ultrices dui. Bibendum arcu
+              vitae elementum curabitur vitae nunc sed velit dignissim. Sed risus pretium quam
+              vulputate dignissim suspendisse in est. Curabitur gravida arcu ac tortor. Nam libero
+              justo laoreet sit amet cursus sit amet. Arcu dui vivamus arcu felis bibendum ut
+              tristique et egestas. Eros donec ac odio tempor orci dapibus ultrices. At erat
+              pellentesque adipiscing commodo elit at. Dignissim cras tincidunt lobortis feugiat
+              vivamus at augue.
+            </p>
+            <p>
+              Amet commodo nulla facilisi nullam vehicula ipsum. Blandit libero volutpat sed cras.
+              Quam lacus suspendisse faucibus interdum posuere. Aenean euismod elementum nisi quis
+              eleifend. Orci nulla pellentesque dignissim enim sit amet venenatis. Diam vel quam
+              elementum pulvinar etiam non quam lacus. Sit amet dictum sit amet justo donec enim
+              diam vulputate. Tincidunt ornare massa eget egestas purus. Pulvinar neque laoreet
+              suspendisse interdum consectetur libero id faucibus. Morbi tincidunt augue interdum
+              velit. Nullam non nisi est sit amet.
+            </p>
+            <p>
+              Aliquet enim tortor at auctor urna nunc id cursus metus. Leo urna molestie at
```

Comment:
Same as above (`marginBottom: 0` to the last paragraph in the `Modal.Body`).
```suggestion
            <p style={{marginBottom: 0}}>
              Aliquet enim tortor at auctor urna nunc id cursus metus. Leo urna molestie at
```